apps/graphql/src/apps/itinerary/helpers/Itineraries.js

```diff
@@ -36,6 +41,17 @@ export const mapVehicle = (type: ?string, uniqueNo: ?string) => ({
   uniqueNo: uniqueNo ?? null,
 });
 
+export const slugsNeedParsing = (location: ?SearchLocation): boolean => {
+  return !!(!location?.ids && location?.slugs);
```

Comment:
This is really hard to read, could it rather be something like 
`return location?.ids == null && location?.slugs != null`?

Reply:
Done!